libs/utils/src/lib/models/operationDiscoveryResults.ts

```diff
@@ -30,6 +30,9 @@ export interface SomeKindOfAzureOperationDiscovery {
   isNotification?: boolean;
   api: OperationApi;
   description: string;
+  operationGroup?: {
```

Comment:
Api is the operation group, why was there a need to add operationGroup explicitly... api is the connector and operation Group is also the connector

Reply:
@preetriti1 we should be adding this logic to the searchservice, until we can move it though I told him he could go ahead and add htis in a way that doesn't impact us. This was an extensibility oversight on our side.

Reply:
I addressed your feedback. Thank you!